### PR TITLE
Updating the TechPreviewNoUpdate list in 4.9

### PR DESCRIPTION
**Pull request by mburke5678**
Updating the list to match https://github.com/openshift/api/blob/release-4.9/config/v1/types_feature.go#L113

Preview: [Understanding feature gates](https://deploy-preview-41639--osdocs.netlify.app/openshift-enterprise/latest/nodes/clusters/nodes-cluster-enabling-features.html#nodes-cluster-enabling-features-about_nodes-cluster-enabling)

### CODE REVIEW

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 1]
```diff
@@ -21,8 +21,18 @@ a|Enables Technology Preview features that are not part of the default features.
...
+** Amazon Web Services (AWS) Elastic Block Storage (EBS)
+** OpenStack Cinder
+** Azure Disk
+** Google Compute Engine Persistent Disk (in-tree)
+** Google Cloud Platform Persistent Disk (CSI).
+* External Cloud Provider. Enables using an external cloud-controller-manager rather than the in-tree cloud controller. Available as a Technology Preview for: 
+** Amazon Web Services (AWS)
+** Microsoft Azure
+** Red Hat OpenStack Platform (RHOSP)
 * `InsightsOperatorPullingSCA`. Enables importing of RHEL Simple Content Access (SCA) certificates from {console-redhat-com}.
```
Comment (tremes) [inline modules/nodes-cluster-enabling-features-about.adoc:36]: Is this really for the 4.9 docs please? If so then this is correct, but if this is for the 4.10 then this lines needs to be removed please.

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 2]
```diff
@@ -21,8 +21,18 @@ a|Enables Technology Preview features that are not part of the default features.
...
+* CSI automatic migration. Enables automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
+** Amazon Web Services (AWS) Elastic Block Storage (EBS)
+** OpenStack Cinder
+** Azure Disk
+** Google Compute Engine Persistent Disk (in-tree)
+** Google Cloud Platform Persistent Disk (CSI).
+* External Cloud Provider. Enables using an external cloud-controller-manager rather than the in-tree cloud controller. Available as a Technology Preview for: 
+** Amazon Web Services (AWS)
+** Microsoft Azure
+** Red Hat OpenStack Platform (RHOSP)
```
Comment (JoelSpeed) [inline modules/nodes-cluster-enabling-features-about.adoc:35]: Google Cloud and vSphere as well for 4.10, though I note this PR says 4.9, are the docs not now on 4.10?

Comment (mburke5678) [reply modules/nodes-cluster-enabling-features-about.adoc:35]: This PR is for 4.9. We neglected to update this list for the last few releases. I have a separate PR for 4.10 feature gate features. Sorry for the confusion.

File: modules/nodes-cluster-enabling-features-about.adoc (hunk 1 above)
Comment (tremes) [reply modules/nodes-cluster-enabling-features-about.adoc:36]: OK thanks. No problem. Then it's correct.

File: modules/nodes-cluster-enabling-features-about.adoc [hunk 3]
```diff
@@ -21,8 +21,18 @@ a|Enables Technology Preview features that are not part of the default features.
...
 
-* link:https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#certificate-rotation[RotateKubeletServerCertificate]. Enables the rotation of the server TLS certificate on the kubelet.
-* link:https://kubernetes.io/docs/concepts/policy/pid-limiting/#pod-pid-limits[Pod PID limits (SupportPodPidsLimit)]. Enables limiting process IDs (PIDs) in pods.
+* Azure Disk CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Microsoft Azure Disk Storage.
+* VMware vSphere CSI Driver Operator. Enables provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
+* CSI automatic migration. Enables automatic migration for supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Supported for:
+** Amazon Web Services (AWS) Elastic Block Storage (EBS)
+** OpenStack Cinder
+** Azure Disk
+** Google Compute Engine Persistent Disk (in-tree)
```
Comment (duanwei33) [inline modules/nodes-cluster-enabling-features-about.adoc:30]: This "Google Compute Engine Persistent Disk (in-tree)" should be removed.

**Review by jeana-redhat — APPROVED** [review]
One question but LGTM

**Comment by bergerhoffer** [comment]
Can't merge to main and CP to 4.9 without going to 4.10 too. CPing!

**Pull request merged**